easybuild/easyconfigs/p/Perl/Perl-5.24.0-foss-2016b-bare.eb

```diff
@@ -0,0 +1,17 @@
+name = 'Perl'
+version = '5.24.0'
+versionsuffix = '-bare'
+
+homepage = 'http://www.perl.org/'
+description = """Larry Wall's Practical Extraction and Report Language"""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'optarch': True, 'pic': True}
```

Comment:
I'll go ahead and merge this as-is (once test reports are in), since a lot of existing easyconfigs already include `'optarch': True`; we'll do a cleanup sweep for that later